erpnext/accounts/report/budget_variance_report/budget_variance_report.py

```
@@ -21,55 +19,262 @@ def execute(filters=None):
+					select
+						name
+					from
+						`tab{filters.budget_against}`
+					where
+						name = gl.{budget_against}
+						{cond}
+				)
+				group by
+					gl.name
+				order by gl.fiscal_year
+		""",
```

Ruff (0.14.10)

Possible SQL injection vector through string-based query construction

(S608)